server/routes/teamEventAPI.js

```diff
@@ -0,0 +1,66 @@
+const express = require("express");
+const db = require("../db/models");
+const auth = require("../middleware/auth");
+
+const router = express.Router();
+
+// 1 - Create a new team event type
+router.post("/api/team-event", auth, (req, res) => {
+  if (req.body.duration) {
+    //
+    db.TeamEventType.create({ ...req.body, teamId: req.teamId })
+      .then((response) => res.send(response))
+      .catch((error) => {
+        console.log(error);
+        res.status(500).send(error);
+      });
+  } else if (!req.body.duration) {
+    res.status(400).send("Duration is required");
+  }
+});
+
+// 2 - Add or remove members from an event type
+// an updated team member is passed to DB to update added and removed members 
+router.post("/api/team/id/:id", (req, res) => {
```

Comment:
```suggestion
router.post("/api/team/:id", (req, res) => {
```
You can probably just keep the id parameter